playbooks/group_vars/webportals.yml

```diff
@@ -118,13 +118,15 @@ webportal_docker_compose_files_mandatory:
 webportal_docker_compose_files_optional:
   - "docker-compose.accounts.yml"
   - "docker-compose.jaeger.yml"
+  - "docker-compose.mongodb.yml"
```

Comment:
mongo should be moved before accounts, because accounts depend on mongo and can possibly override it.